### PR TITLE
DRAFT: RFC-0002

### PR DESCRIPTION
**Pull request by MarkBennett**
This RFC creates a new process by which Dev Edmonton Society members can create new working groups. It's objective is to open up the work of the society to all members, allowing members of all skills, interests, and availability to contribute and shape their community.

It also includes basic scaffolding to create a list of RFCs on the Jekyll website. The styling and layout is all placeholder, so please view this Markdown via Github for now.

### CODE REVIEW

**Review by MarkBennett — COMMENTED** [review]
Ready for sharing! :smile:

**Comment by benzittlau** [comment]
Should we be presenting guidance on member policy for these working groups?  

Separate out the list of working groups from the RFC document, so changes to status of working groups don't need amendments to this doc.

File: _rfcs/rfc-0002.md [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+The Society reminds all Working Groups that messages on Slack are not permenant and everything posted there will disappear after a month or so. If recording a disucssion or decision is important, then Slack is not the place to record it.
+
+# Current Working Groups
+
+| Working Group Name | Mandate | Champion | Website | Mailing List | Slack Channel | Meetings |
+|--------------------|---------|---------|---------|--------------|---------------|---|
+| Board of Governers | Provide oversight and strategic direction to DES. | Mark Bennett | ?? | ?? | #dev-edmonton-society | bi-monthly, per calendar |
+| Event Recording    | Allow event hosts to record and share events. | Aaron Yong | ?? | ?? | #event-recording | none |
+| Meetup Support | Provide resources, materials, and services to make running meetups and events easier. | ?? | | | | bi-monthly, per calendar |
+| Finance | Contol and report on DES finances | Estyn Edwards | | | | monthly |
```
Comment (benzittlau) [inline _rfcs/rfc-0002.md]: Contol -> Control

**Comment by MarkBennett** [comment]
> Should we be presenting guidance on member policy for these working groups?
> 
> Separate out the list of working groups from the RFC document, so changes to status of working groups don't need amendments to this doc.

@benzittlau can we circle back to this in August? For now, it's simpler to keep them here as the document is changing anyway, but going forward it might make sense to list these somewhere else. Another RFC might be a reasonable place to do that TBH as that would allow other members to submit PRs to create new Working Groups.

I'll add this to the outstanding issues.

**Comment by MarkBennett** [comment]
As per the most recent AGM, this process has been accepted into draft for review by the Board in August 2020. It's assumed that outstanding issues will be addressed by that time, inline with this being accepted formally then.

In the main time this will be published as a draft and implemented by DES as is, so that practical feedback can be collected and this process revised.

**Pull request merged**